plutus-tx/src/Language/PlutusTx/AssocMap.hs

```diff
@@ -38,6 +40,8 @@ newtype Map k v = Map { unMap :: [(k, v)] }
     deriving stock (Generic)
     deriving newtype (Eq, Ord)
 
+deriving anyclass instance (Typeable k, Typeable v) => ToSchema (Map k v)
```

Comment:
I'd rather have this as an orphan instance than make `plutus-tx` depend on `playground-common`. We already have a bunch of orphan instances for `plutus-tx` types in `plutus-wallet-api`.

Reply:
Ah, okay. Fair enough. Will do. 👍